style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  outline: none;
+  border: none;
+  text-decoration: none;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.wrap {
+  height: 718px;
+  width: 375px;
+  border: 2px solid #ccc;
+  border-radius: 0.5em;
+  padding: 25px 25px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.headr {
+  margin-bottom: 20px;
+}
+
+.headr .flex {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 1.5rem;
+}
+
+.headr .flex .logo {
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 700;
+}
+
+.headr .flex .bars {
+  font-size: 3rem;
+  cursor: pointer;
+  color: #6070ff;
+}
+
+/* headlin */
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 24px;
+  background-image: url(unt/header-shapes\ mobile.png);
+  gap: 12px;
+}
```

Comment:
- [ ] The `background image` is not properly aligned and it does not look like what we have in the Figma design, Kindly see the container you are assigning the background image to and make amendments so that your design can be as perfect as what we have in the Figma design.

Your design 👇 

![john](https://user-images.githubusercontent.com/78634841/211955831-b3105500-6564-4176-9007-bbc011744855.png)

Figma design 👇 

![real](https://user-images.githubusercontent.com/78634841/211955944-48618eb2-638d-438c-9105-aa5cdefb45ad.png)